docs/framework/wcf/feature-details/large-data-and-streaming.md

```diff
@@ -62,7 +62,7 @@ manager: "erikre"
   
  For data that does not have these constraints, it is typically better to send sequences of messages within the scope of a session than one large message. [!INCLUDE[crdefault](../../../../includes/crdefault-md.md)] the "Streaming Data" section later in this topic.  
   
- When sending large amounts of data you will need to set the maxAllowedContentLength IIS setting (for more information see [Configuring IIS Request Limits](http://go.microsoft.com/fwlink/?LinkId=253165)) and the maxReceivedMessageSize binding setting (for example <xref:System.ServiceModel.BasicHttpBinding.MaxReceivedMessageSize%2A> or <xref:System.ServiceModel.NetTcpBinding.MaxReceivedMessageSize%2A>). The maxAllowedContentLength property defaults to 28.6 M and the maxReceivedMessageSize property defaults to 64KB.  
+ When sending large amounts of data you will need to set the maxAllowedContentLength IIS setting (for more information see [Configuring IIS Request Limits](http://go.microsoft.com/fwlink/?LinkId=253165)) and the maxReceivedMessageSize binding setting (for example [System.ServiceModel.BasicHttpBinding.MaxReceivedMessageSize](xref:System.ServiceModel.HttpBindingBase.MaxReceivedMessageSize%2A) or <xref:System.ServiceModel.NetTcpBinding.MaxReceivedMessageSize%2A>). The maxAllowedContentLength property defaults to 28.6 M and the maxReceivedMessageSize property defaults to 64KB.  
```

Comment:
Since these are configuration settings, they should be fenced:
`maxAllowedContentLength` and `maxReceivedMessageSize`